framework/Util/TFirebugLogRoute.php

```diff
@@ -87,25 +84,25 @@ protected function renderHeader()
 		return '';
 	}
 
-	protected function renderMessage($log, $info)
+	protected function renderMessage($log, $meta)
```

Comment:
The TFirebugLogRoute should probably turn off the prefix message with a custom formatLogMessage.  That would be easiest rather than trying to figure out the prefix fields, especially since the prefix can be customized by the application.

Reply:
I agree, better remove the prefix: time ip user and session are not that useful when in a browser console